ImageFeed/Profile/ProfileViewController.swift

```diff
@@ -0,0 +1,84 @@
+import UIKit
+
+final class ProfileViewController: UIViewController {
+    
+    private var userImage: UIImageView!
+    private var usernameLabel: UILabel!
+    private var loginLabel: UILabel!
+    private var statusLabel: UILabel!
+    private var logoutButton: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addUserImage()
+        addUsernameLabel()
+        addLoginLabel()
+        addStatusLabel()
+        addLogoutButton()
+        
+    }
+    
+    func addUserImage() {
+        let placeholder = UIImage(named: "UserImagePlaceholder")
+        userImage = UIImageView(image: placeholder)
+        userImage.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(userImage)
+        NSLayoutConstraint.activate([
+        userImage.heightAnchor.constraint(equalToConstant: 70),
+        userImage.widthAnchor.constraint(equalToConstant: 70),
+        userImage.topAnchor.constraint(equalTo: view.topAnchor, constant: 76),
+        userImage.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 16)
+        ])
```

Comment:
Очень хорошо что настроил констрейнты за один вызов! это оптимальное решение

Отступы думаю вот так правильнее оставить. не забывай использовать на выделенном коде control+i пока не набьешь руку
```suggestion
        NSLayoutConstraint.activate([
            userImage.heightAnchor.constraint(equalToConstant: 70),
            userImage.widthAnchor.constraint(equalToConstant: 70),
            userImage.topAnchor.constraint(equalTo: view.topAnchor, constant: 76),
            userImage.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 16)
        ])
```

Reply:
можно придерживаться этого правила и дальше